.github/workflows/IntegrationTest.yml

```diff
@@ -54,10 +54,11 @@ jobs:
             end
           end
           try
+            Pkg.develop(PackageSpec(path="downstream"))
             # force it to use this PR's version of the package
             Pkg.develop(PackageSpec(path="."))  # resolver may fail with main deps
             Pkg.update()
-            Pkg.test(coverage=true)  # resolver may fail with test time deps
+            Pkg.test("MapBroadcast")  # resolver may fail with test time deps
```

Comment:
This does not look right though, you still need to extract the name of the downstream package somehow

Reply:
Yes, I just noticed that an pushed a fix.